### ISSUE TITLE
4.4.3 FOLLOW(S) issue

### ISSUE THREAD

**Issue by x-ji** [fool2fish/dragon-book-exercise-answers#47]
Should be {$, a, +, *} instead of {a, +, *}

**Comment by sudheesh001** [comment]
Precisely, I think this change has to be made. @fool2fish

**Comment by NoteBookie** [comment]
what's the reason for 'a' to stay in FOLLOW(S)?  If it does, you will get a conflict at M[A, a].

**Comment by NoteBookie** [comment]
S -> a A
A-> S + A | S \* A | ε
(I thought it worked.)

**Comment by zackszhu** [comment]
@NoteBookie yeah 'a' should be removed

**Comment by Yudezhi** [comment]
No! I think 'a' should be put in,it's what the question asked.the @NoteBookie  did is  how to fix it.Here what we should do is get the follow set by the rule.

**Comment by theo-one** [comment]
'a'   shouldn't  be in FOLLOW(s) @Yudezhi
